src/jwt/jwt.ts

```diff
@@ -29,7 +29,7 @@ export function generateJWT<T extends object>(
         throw new ServerError('INVALID CONFIGURATION', 'Missing secret for JWT generation')
     }
 
-    return sign(payload, asymetric ? process.env.JWT_PRIVATE_KEY : process.env.NEXTAUTH_SECRET, {
+    return sign(payload, Buffer.from(asymetric ? process.env.JWT_PRIVATE_KEY : process.env.NEXTAUTH_SECRET), {
```

Comment:
It isn't obvious, at least for me, what the option "asymetric" does, the docstring should probably be updated.